src/content/utilities/loading/loading.md

```diff
@@ -0,0 +1,128 @@
+
+There are several components that should **never** be represented by a Skeleton State since the data for these items should already be loaded in by the time a user interacts with them. These include:
+
+- Toast Notification
+- Overflow Menu
+- Dropdown Items
+- Modal (elements inside a modal may have a Skeleton State, however the Modal itself should not be skeletonized)
+- Small and Large Loader (used for other loading cases but not Skeleton States)
+
+
+### Progressive loading
+
+When utilizing Skeleton States, the order in which components, data, and content load in needs to be planned and designed for using the progressive loading technique. Progressive loading is when content loads in batches. The first batch to load should be the most simplified view of the page, followed by the secondary and tertiary batches, with each batch adding more page detail until the entire viewport has been loaded. The primary batch should illustrate a page's structural layout (the Skeleton State versions of the container-based components), data-based text (the Skeleton State version of text) and non-data text. Secondary and tertiary batches can include images, content below the fold, interactive (action-based) components, and/or data-based text. Remember that not all items need a Skeleton State, but instead can be expressed as negative or white space until they load in. For example a 600 x 600px image could be shown as a 600 x 600px area of white space until the secondary or tertiary batch when the full image loads in.
+
+![Example of a dashboard using Skeleton States, to load in with progressive loading over three screens.](images/Progressive-Loading.png)
+_Example of a dashboard using Skeleton States to demonstrate progressive loading_
+
+
+
+### Code
+
+The skeleton state for _most_ components can be displayed simply by adding a modifier class of `bx--skeleton` to the component while data is loading and then removing the class after it has loaded in. You will have to create your own custom javascript to implement this based on your environment.
+
+```
+
+/* Loading */
+<button class="bx--btn bx--btn--primary bx--skeleton"></button>
+
+/* Loaded */
+<button class="bx--btn bx--btn--primary">Button loaded</button>
+
+```
+
+If you are using our [React library](http://react.carbondesignsystem.com/) there are separate skeleton state components available to display while data is loading. View an [example](https://codesandbox.io/s/wq264y43k8) of this for React.
+
+```
+const DataComponent = ({ loading, data }) => {
+  if (loading) {
+    return <CarbonComponent.skeleton />
+  }
+  return <CarbonComponent field1={data.field1} />
+}
+```
+
+| Component          | Vanilla | React |
+|--------------------|---------|-------|
+| Accordion          |  | ✅|
```

Comment:
@marijohannessen Is there a way to add inline svgs with markdown currently in the site? Or should we just use an image tag here?